### ISSUE TITLE
System Tray Icon Disappears

### ISSUE THREAD

**Issue by aguaagua2** [Ross-Patterson/Portals-Desktop-Organization#189]
**Describe the bug**
I installed Portals and got it parameterized and working but when I want to re-modify parameters, I can't get Portal interface to show up.

**Expected behavior**
Each time I would need to change parameters, I should be able to get the interface to show up

**Desktop (please complete the following information):**
 - Windows 7
 - Opera 95.0.4635.84
 - 3.0.2.2

**Comment by aguaagua2** [comment]
I also noticed that the known bug of Portals windows disappearing when clicking on "Show Desktop" button seems to be tied to the presence or absence of Portals in the System Tray.  Portals.exe may be running but if for any reason Windows Explorer has blown away Portals icon in the tray, the disappearing bug enters in, yet when Portals icon is stil active in the tray, "Show Desktop" does not make the Portals windows disappear on Desktop.

**Comment by Ross-Patterson** [comment]
The interface does not show on start of the app if you already have some portals created. This is intentional because it would be really annoying if every time you started your PC it threw up that window.

You can always get it back by just clicking the icon in the System tray.

See here:
https://portals-app.help/help/installation-setup/opening-the-main-window/

If the icon is not in the system tray, Portals is not running.

**Comment by aguaagua2** [comment]
Hi,

Thanks for your reply.  I understand the hot showing up on start of the app, it makes sense.

Now I just want to say that there is the case where the system tray icon may have disappeared while the app is still running (I can see it with Task Manager, while (sometimes, probably after some Explorer failure) no more its icon in system tray icon list yet)).  And in that case, if I want to change any Portals parameter, I only have the solution of killing its process and relaunching it, in order to have the interface appear.

If there is any other better way to get the interface appear, please let me know.

**Comment by Ross-Patterson** [comment]
I have never ever seen or heard of the tray icon not showing.

Are you sure it hasn't just gone into the "Hidden Icons" section:
![image](https://github.com/Ross-Patterson/Portals-Desktop-Organization/assets/66630787/3ac6e581-e0ad-407c-83ba-22b324e41575)

Can you send a screengrab of your task manager showing Portals.exe running, with the taskbar shown, including showing the "Hidden Icons" section.

**Comment by aguaagua2** [comment]
OK, I'll do that next time this occurs.  
But I am pretty sure as when that happened, I went on "Control Panel\All Control Panel Items\Notification Area Icons", made sure all items were in "Show icons and notifications" and Portals was not there, while Task Manager was show Portals.exe running.
But as I said, it's very probably an aberration of MS Explorer crashing partially or something.

**Comment by aguaagua2** [comment]
I sent you by e-mail the screengrabs you mentionned, as this issue happened again recently.  Let me know if it helps.

**Comment by Ross-Patterson** [comment]
I didn't receive anything unfortunately. You can send them to ross@rosspat.dev

Or just upload them here.

**Comment by aguaagua2** [comment]
Here is attached the file "Portals Issue 20231101.zip" which contains the screengrabs.
[Portals Issue 20231101.zip](https://github.com/Ross-Patterson/Portals-Desktop-Organization/files/13273351/Portals.Issue.20231101.zip)

**Comment by Ross-Patterson** [comment]
Have you checked here:
![image](https://github.com/Ross-Patterson/Portals-Desktop-Organization/assets/66630787/4674e847-eae9-4502-99d3-5446f179d22f)

**Comment by aguaagua2** [comment]
Yes, I did.  And the details or list of which System Tray icons were present is one of the screengrabs I attached in my previous message, namely "3-System Tray icons...".

**Comment by Ross-Patterson** [comment]
I am still not able to replicate at all - which version of windows are you using, are you using any other apps to customize windows?

**Comment by Ross-Patterson** [comment]
closing due to no response
